doc/install_mne_python.rst

```diff
@@ -62,6 +62,9 @@ Install Python and MNE-Python
                   <pre><span></span><span class="gp">$</span> curl -O https://raw.githubusercontent.com/mne-tools/mne-python/master/environment.yml<br><span class="gp">$</span> conda env create -f environment.yml<br><span class="gp">$</span> source activate mne</pre>
                 </div>
               </div>
+              <p>If Mayavi plotting in Jupyter Notebooks doesn't work well, using the IPython magic "%gui qt" after importing MNE/Mayavi/PySurfer may 
```

Comment:
Looks fine to me except the `%gui qt` should be changed to look like inline code. Can you see if `<span class="gp">...</span>` or `class="pre"` will make it look like inline code?